lib/document/stateTransition/DocumentsBatchTransition.js

```diff
@@ -0,0 +1,95 @@
+const AbstractStateTransition = require('../../stateTransition/AbstractStateTransition');
+const stateTransitionTypes = require('../../stateTransition/stateTransitionTypes');
+
+const AbstractDocumentTransition = require('./documentTransition/AbstractDocumentTransition');
+const DocumentCreateTransition = require('./documentTransition/DocumentCreateTransition');
+const DocumentReplaceTransition = require('./documentTransition/DocumentReplaceTransition');
+const DocumentDeleteTransition = require('./documentTransition/DocumentDeleteTransition');
+
+const actionsToClasses = {
+  [AbstractDocumentTransition.ACTIONS.CREATE]: DocumentCreateTransition,
+  [AbstractDocumentTransition.ACTIONS.REPLACE]: DocumentReplaceTransition,
+  [AbstractDocumentTransition.ACTIONS.DELETE]: DocumentDeleteTransition,
+};
+
+class DocumentsBatchTransition extends AbstractStateTransition {
+  /**
+   * @param {RawDocumentsBatchTransition} rawStateTransition
+   */
+  constructor(rawStateTransition) {
```

Comment:
I guess we should allow to create empty ST.

Reply:
@shumkov why?

Reply:
We do that in IdentityCreateTransition and abstract state transition classes.